themes/survey/fruity_twentythree/css/variations/theme_blueberry.css

```
@@ -15511,122 +15511,323 @@ body .imageselect-listitem label::after {
 }
 
-/*
-Max width before this PARTICULAR table gets nasty
-For iPads specifically : add this : (min-device-width: 768px) and (max-device-width: 1024px)
-*/
-@media only screen and (max-width: 760px), (min-device-width: 768px) and (max-device-width: 1024px) {
-  table.ls-answers, table.ls-answers thead, table.ls-answers tbody, table.ls-answers th, table.ls-answers td, table.ls-answers tr {
+/* Fix for IE11 - no array question radio buttons on small screens */
+@media all and (max-width: 760px) and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+  table.ls-answers .checkbox-item label.ls-label-xs-visibility::before {
+    margin-left: 0px;
```

Stylelint (17.2.0)

[error] Unexpected unit (length-zero-no-unit)

(length-zero-no-unit)